lib/gradient.py

```diff
@@ -0,0 +1,37 @@
+from color import Color, interpolate_colors
+
+from lib.utils import interpolate_value
+
+class Gradient(object):
+	def __init__(
+		self,
+		seconds: float,
```

Comment:
What is `seconds`? It would be nice to have a comment here on what this is measuring -- and maybe some validation?

Reply:
generally it's a time associated with the gradient. Sometimes that is time of day in the case of the spec file, sometimes that's unix time in the case of fading. Definitely don't like it dual use like that though